apps/earthbound/.env.sample

```
@@ -1,7 +1,8 @@
 NEXT_PUBLIC_CDN_URL=https://cdn.oasa.co/photo/
 # NEXT_PUBLIC_API_URL=http://localhost:8080
 NEXT_PUBLIC_API_URL=https://dev.api.project.com
-NEXT_PUBLIC_PLATFORM=
+NEXT_PUBLIC_APP_NAME=earthbound
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_APP_NAME key should go before the NEXT_PUBLIC_CDN_URL key

(UnorderedKey)